ext/drwrap/drwrap.h

```diff
@@ -887,6 +889,18 @@ DR_EXPORT
 bool
 drwrap_get_stats(INOUT drwrap_stats_t *stats);
 
+DR_EXPORT
+/**
+ * If the provided app_pc (\p possibly_sentinel) is indeed the address of the
+ * internal replace_retaddr_sentinel(), this routine replaces it with the actual
```

Comment:
replace_retaddr_sentinel is not a public symbol and is not mentioned anywhere else in public docs: probably better to just say "the return address sentinel used to implement #DRWRAP_REPLACE_RETADDR".  Ditto below.

Reply:
Done.